src/TicketService.java

```diff
@@ -27,6 +35,12 @@ public static void main(String[] args) {
         Ticket limitedTicket = new Ticket("Sphear", 122, System.currentTimeMillis());
         ticketService.addTicket(limitedTicket);
 
+        // Example usage of getTicketsByEventCode
+        List<Ticket> eventTickets = ticketService.getTicketsByEventCode(111);
+        System.out.println("Tickets with event code 111:");
```

Comment:
Using System.out.println (often abbreviated as Sysout) for logging in production code is generally not recommende

Reply:
Thank you for your review! Well understood!